sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -927,6 +931,12 @@ public partial class ServerCertificateCustomValidationArgs
         public System.Security.Cryptography.X509Certificates.X509Chain? CertificateAuthorityChain { get { throw null; } }
         public System.Net.Security.SslPolicyErrors SslPolicyErrors { get { throw null; } }
     }
+    public partial class TelemetryPackageInfo
+    {
+        internal TelemetryPackageInfo() { }
+        public string UserAgentValue { get { throw null; } }
+        public static Azure.Core.Pipeline.TelemetryPackageInfo Create<T>(string? applicationId = null) { throw null; }
```

Comment:
I don't feel strongly about it - but I liked making the Create method generic to avoid passing a type explicitly. This could be done by making the whole type generic as well.

Reply:
Changed to names.